packages/template-retail-react-app/app/pages/account/order-history.jsx

```diff
@@ -99,14 +99,20 @@ const AccountOrderHistory = () => {
 
     const pageUrls = usePageUrls({total: paging.total, limit})
 
+    const headingRef = useRef()
+    useEffect(() => {
+        // Focus the 'Order History' header when the component mounts for accessibility
+        headingRef?.current?.focus()
+    }, [])
+
     useEffect(() => {
         window.scrollTo(0, 0)
     }, [customer, searchParams.offset])
 
     return (
         <Stack spacing={4} data-testid="account-order-history-page">
             <Stack>
-                <Heading as="h1" fontSize="2xl">
+                <Heading as="h1" fontSize="2xl" tabIndex="0" ref={headingRef}>
```

Comment:
It looks like we forgot to implement the same solution on the Order details page:

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/71c8cceeb4345af233061639e971424db007fea7/packages/template-retail-react-app/app/pages/account/order-detail.jsx#L151